addons/zylann.scatter/tools/util.gd

```diff
@@ -3,7 +3,7 @@ static func get_scene_aabb(node, aabb=AABB()):
 	if not node.visible:
 		return aabb
 	if node is VisualInstance:
-		var node_aabb = node.global_transform.xform(node.get_aabb())
+		var node_aabb = node.transform.xform(node.get_aabb())
```

Comment:
`transform` is a local transform. If a part of the scene is scaled by its parent, it wont be taken into account. But at the same time, Godot is overly pedantic when it comes to accessing "global transform" before adding to the tree, and I dont like that... if we really want to use `transform` then we need to change more things.